### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by pdhoopr**
This PR sets up the foundation of the project with a few pieces of tech:

- [Parcel](https://parceljs.org/)
- [Babel](http://babeljs.io/)
- [ESLint](https://eslint.org/)
- [Flow](https://flow.org/)
- [Sass](http://sass-lang.com/) + [Autoprefixer](https://github.com/postcss/autoprefixer)
- [Stylelint](https://stylelint.io/)

Totally open to suggestions about how things are configured or anything else we might want to add!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,48 @@
...
+
+## Usage
+
+Once all the setup is complete, you can actually start using and developing the app:
+
+- Start the app's development server using `yarn run start`, then go to `http://localhost:1234` in your browser to see it (stop the server with the `Ctrl` + `C` keys)
+- If you want to run the linters on your code, use `yarn run lint`
+  - To just lint your JavaScript code, `yarn run lint:js`
+  - To just lint your CSS/Sass code, `yarn run lint:sass`
+- If you want to run Flow's type checks, use `yarn run flow`
```
Comment (cbothner) [inline README.md]: Isn’t `run` usually left out when using Yarn? `yarn start` etc. work just fine

File: README.md [hunk 2]
```diff
@@ -1 +1,48 @@
...
+Getting the app installed isn't too scary! Just a few simple steps and you'll be all set:
+
+- Create an empty repo in your own GitHub account to house your copy of the app
+- Open a terminal/console and navigate to whatever folder you want the project to be in
+- Clone the sample repo with `git clone https://github.com/letscodework/email-client-practice-app.git`
+- Navigate into the downloaded folder with `cd email-client-practice-app`
+- Associate the Git info with yourself:
+  - `rm -rf .git`
+  - `git init`
+  - `git remote add origin YOUR_PROJECT_REPOSITORY_URL`
```
Comment (cbothner) [inline README.md]: Why not suggest they fork this repo instead of having to reinit `.git` and change remotes?

**Review by cbothner — COMMENTED** [review]
This looks like a great setup. Very easy to use, and supports everything I expect it to out of the box (except sourcemaps which [are coming](https://github.com/parcel-bundler/parcel/pull/506) and will hopefully land before we get started).

File: README.md (hunk 2 above)
Comment (pdhoopr) [reply README.md]: Good point. I wasn't sure if we wanted them to inherit the Git history for the project so it seems more like their own, but forking seems easier. Updated to be forking instead.

File: README.md (hunk 1 above)
Comment (pdhoopr) [reply README.md]: 💥 Mind blown! I thought it just followed npm where only some of them worked w/o `run`. Fixed!

**Review by mariehooper — APPROVED** [review]
This looks good to me!

**Pull request merged**